board/src/main/java/com/kdt/prgrms/board/dto/PostAddRequest.java

```diff
@@ -0,0 +1,34 @@
+package com.kdt.prgrms.board.dto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+public class PostAddRequest {
+
+    @NotBlank
+    @NotNull
+    private final String title;
+
+    @NotBlank
+    @NotNull
+    private final String content;
```

Comment:
@NotNull은 @NotBlank에 포함되어 있는 걸로 알고 있습니다! 불필요한 Annotation 사용일 것 같습니다!
```
The annotated element must not be null and must contain at least one non-whitespace character. Accepts CharSequence.
Since: 2.0
See Also: Character.isWhitespace(char)
Author: Hardy Ferentschik
```

Reply:
넵 감사합니다! 수정하겠습니다!